src/web/mjs/connectors/AssortedScans.mjs

```diff
@@ -0,0 +1,65 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs'
+
+export default class AssortedScans extends Connector {
+    constructor() {
+        super();
+        super.id = 'assortedscans';
+        super.label = 'assortedscans';
+        this.tags = ['manga', 'english'];
+        this.url = 'https://assortedscans.com/';
+    }
+
+    async _getMangaFromURI(uri) {
+        let request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, '#series-title');
+        let title = data.text.trim();
+        let id = uri.pathname + uri.search;
+        return Manga(this, id, title);
+    }
+
+    async _getMangas() {
+        let request = new Request(new URL('/reader/', this.url), this.requestOptions);
+        let data = await this.fetchDOM(request, 'section.series h2.series-title a');
+        return data.map(element => {
+                return {
+                    id: element.href,
+                    title: element.text.trim(),
+                };
+            }
+        )
+    }
+
+    async _getChapters(manga) {
+        let id = manga.id.match('/reader/.*/?')[0];
+        let request = new Request(new URL(id, this.url), this.requestOptions);
+        let data = await this.fetchDOM(request, 'div.chapter a');
+        return data
+            .filter((_, i) => (i + 1) % 2) // Every second chapter link is a link to the translator
+            .map(element => {
+                return {
+                    id: element.href,
```

Comment:
Use `this.getRootRelativeOrAbsoluteLink(element, this.url)` to get relative path